src/main/java/com/enterprise/fundraisermanager/FundraiserManagerBaseController.java

```diff
@@ -0,0 +1,9 @@
+package com.enterprise.fundraisermanager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class FundraiserManagerBaseController {
+
+    Logger logger = LoggerFactory.getLogger(FundraiserManagerBaseController.class);
```

Comment:
Added a base controller class where shared code can be added. Also added a logger because I though it could be shared between controllers

Reply:
Is this used anywhere?

This appears to be an incomplete feature.
Only features that meet the team's Definition of Done should be merged to master.
Otherwise, it is clutter, which is technical debt.